PublishedModules/Psychokiller1888/Minigames/Minigames.py

```diff
@@ -50,8 +50,7 @@ def __init__(self):
 				lib = importlib.import_module(f'modules.Minigames.model.{game}')
 				klass = getattr(lib, game)
 				minigame = klass()
-				self._minigames[game] = minigame
-				self._SUPPORTED_INTENTS += minigame.intents
+				self._INTENTS = {**self._INTENTS, **dict.fromkeys(minigame.intents, self.minigameIntent)}
```

Comment:
how does alice know about these intents, when they are added to the variable after it got passed to the parent